File: Json.Tests/assets/Schema/Typescript/UserStore/Friflo.Json.Flow.UserAuth.ts
import { Right } from "./Friflo.Json.Flow.Auth.Rights"
import { Right_Union } from "./Friflo.Json.Flow.Auth.Rights"

export class Role {
    id:     string;
    rights: Right_Union[];
}

export class UserCredential {
    id:       string;
    passHash: string;
    token:    string;
}

File: Json.Tests/assets/Schema/Kotlin/src/main/kotlin/UserStore/Friflo.Json.Flow.UserAuth.kt

data class Role (
    val rights       : List<Right>,
    val description? : String | null,
)

data class UserCredential (
    val passHash? : String | null,
    val token?    : String | null,
)

data class UserPermission (
    val roles? : List<String> | null,
)

